core/src/main/java/ch/cyberduck/core/vault/registry/VaultRegistryFindFeature.java

```diff
@@ -57,11 +57,12 @@ public boolean find(final Path file, final ListProgressListener listener) throws
         if(vault.equals(Vault.DISABLED)) {
             if(autodetect) {
                 final Path directory = file.getParent();
-                final Path key = new Path(directory,
-                        new HostPreferences(session.getHost()).getProperty("cryptomator.vault.masterkey.filename"), EnumSet.of(Path.Type.file));
-                if(proxy.find(key, listener)) {
+                final Path vaultConfig = new Path(directory,
```

Comment:
Not going to work for vaults that do not have a `vault.cryptomator` file which was introduced only recently.